website/docs/r/compute_instance_group.html.markdown

```diff
@@ -97,3 +97,11 @@ exported:
 * `self_link` - The URI of the created resource.
 
 * `size` - The number of instances in the group.
+
+## Import
+
+Instance group can be imported using the `zone` and `name`, e.g.
```

Comment:
I decided to import using zone/name because it is a valid case to have multiple instance groups with the same name across different zones. 

I will sync up with @danawillow to ensure consistency on our approach for all importable resources.